update/update.go

```diff
@@ -20,9 +20,8 @@ import (
 // EncodeJSON returns JSON (as bytes) for an update
 func EncodeJSON(version string, name string, description string, src string, URI fmt.Stringer, signature string) ([]byte, error) {
```

Comment:
Would you like to change `signature` to `signaturePath` too, then, for consistency?